src/browser_action/core.css

```diff
@@ -142,29 +142,29 @@ h1 {
 }
 
 /* tabs */
-.result-tabs {
+.web-vitals-chrome-extension-popup .result-tabs {
     color: #fff;
     font-weight: 500;
     height: 50px;
     left: 0;
     right: 0;
 }
 
-.result-tabs .goog-tab-selected .icon, .result-tabs .goog-tab-selected.goog-tab-hover .icon, .result-tabs .goog-tab-selected .tab-title, .result-tabs .goog-tab-selected.goog-tab-hover .tab-title {
+.web-vitals-chrome-extension-popup .result-tabs .goog-tab-selected .icon, .result-tabs .goog-tab-selected.goog-tab-hover .icon, .result-tabs .goog-tab-selected .tab-title, .result-tabs .goog-tab-selected.goog-tab-hover .tab-title {
```

Comment:
Add the scope to all selectors.

```suggestion
.web-vitals-chrome-extension-popup .result-tabs .goog-tab-selected .icon,
.web-vitals-chrome-extension-popup .result-tabs .goog-tab-selected.goog-tab-hover .icon,
.web-vitals-chrome-extension-popup .result-tabs .goog-tab-selected .tab-title,
.web-vitals-chrome-extension-popup .result-tabs .goog-tab-selected.goog-tab-hover .tab-title {
```

Reply:
Oh, good catch!

Reply:
Next commit should address this.